docs/source/conf.py

```diff
@@ -0,0 +1,99 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# pylint: disable=W,C
+import os
+import sys
+import toml
+
+sys.path.insert(0, os.path.abspath("../.."))
+toml_dict = toml.load("../../pyproject.toml")
+
+
+# -- Project information -----------------------------------------------------
+
+project = toml_dict["tool"]["poetry"]["name"]
+copyright = f"{toml_dict['tool']['poetry']['authors'][0]}, 2021"
+author = ",".join(toml_dict["tool"]["poetry"]["authors"])
+
+# The full version, including alpha/beta/rc tags
+release = toml_dict["tool"]["poetry"]["version"]
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.napoleon", "m2r2"]
+
+autodoc_default_options = {
+    # "exclude-members": "__weakref__",
+    # "ignore-module-all": ,
+    # "imported-members": ,
+    # "inherited-members": ,
+    "members": True,
+    # "member-order": "bysource",
+    # "private-members": ,
```

Comment:
Does it make sense to keep these?, being that they're just commented out?

Reply:
Took them out. Good call. I believe Sphinx makes it that way when you start with it's template. Thanks Jeff.